### PR TITLE
Write more documentation

### PR DESCRIPTION
**Pull request by i-like-robots**
This PR fleshes out and ties together some of the documentation we have so far accumulated. I have gone through all of the items included in the documentation navigation.

### CODE REVIEW

File: docs/get-started/installation.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Installing x-dash
+
+The x-dash repository contains everything you need to w
```
Comment (quarterto) [inline docs/get-started/installation.md]: where's the rest of this s

File: docs/get-started/what-is-x-dash.md [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+The primary goal is to create a set of UI components that are shareable between The App and FT.com. We aim to provide the patterns and tools required for developers to create high quality code which is suitable for use across both products.
+
+We want to do this to provide a consistent experience for our users who move between the two.
+
+
+## Why is this so hard?
+
+Making components for both The App and FT.com is a technical challenge because the two products have different tech stacks, different architectures, and a different history. This means we must find a new middleground without introducing compromises or new technolgies which might prevent teams from adopting it.
+
+This project is also a tricky cultural challenge because the two products have different life-cycles — the website ships many times a day, whereas an app may remain on a users phone for several months — and not all existing components may have been originally conceived with both products in mind meaning we are unable to "lift and shift" them.
```
Comment (quarterto) [inline docs/get-started/what-is-x-dash.md]: `s/users/user's/`

File: docs/components/javascript.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# Component JavaScript
+
+
+## Architecture
+
+Each component is a function which accepts an object containing all of the properties needed to render it and returns the elements describing what to render. These are called functional components or stateless functional components by many frameworks.
+
+Following this standard pattern means that components authored with x-dash can be compatible with a variety of static and dynamic runtimes including React, Preact, Inferno, VDO, and Hyperons amongst others.
+
+Interactive components which react to state changes can still be created using the [x-interaction] component.
```
Comment (quarterto) [inline docs/components/javascript.md]: > react to state changes

a more accurate wording would be "change state in reaction to events"?

File: docs/get-started/what-is-x-dash.md [hunk 4]
```diff
@@ -1,7 +1,44 @@
...
+## What are goals of this project?
+
+The primary goal is to create a set of UI components that are shareable between The App and FT.com. We aim to provide the patterns and tools required for developers to create high quality code which is suitable for use across both products.
+
+We want to do this to provide a consistent experience for our users who move between the two and act as an aid the technical evolution of both products.
+
+
+## Why is this a challenge?
+
+Making components for both The App and FT.com is a technical challenge because the two products have different tech stacks, different architectures, and a different history. This means we must find a new middleground without introducing compromises or new technolgies which might prevent teams from adopting it.
```
Comment (benbarnett) [inline docs/get-started/what-is-x-dash.md]: `technolgies`

File: docs/components/stories.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+	'o-typography': '^5.5.0'
+};
+
+// Story configuration modules
+exports.stories = [
+	require('./food')
+];
+
+// A module generating dynamically editable knobs
+exports.knobs = require('./knobs');
```
Comment (benbarnett) [inline docs/components/stories.md]: Worth mentioning the [fetch-mock stuff](https://github.com/Financial-Times/x-dash/pull/109/files) here too? Or too niche?

File: docs/components/javascript.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+Following this pattern means that components authored with x-dash can be compatible with a variety of static and dynamic runtimes including React, Preact, Inferno, VDO, and Hyperons amongst others.
+
+Interactive components which change state in reaction to events can still be created using the [x-interaction] component.
+
+[x-interaction]: ../../components/x-interaction
+
+
+## Syntax
+
+You may use all syntax up to and including the ECMAScript 2018 specification and any features that may be polyfilled by the [Polyfill Service] in your component source code. We do not currently support transpiling any features at proposal, draft, or candidate stages (if you are unsure what this means check out the [TC39 Process document]).
```
Comment (benbarnett) [inline docs/components/javascript.md]: Could you elaborate on how to use the Polyfill Service? Can we assume it is available? Or should components indicate that they are relying on certain features? This discussion and more over here: https://github.com/Financial-Times/x-dash/pull/78#issuecomment-424346708

File: docs/components/stories.md (hunk 5 above)
Comment (i-like-robots) [reply docs/components/stories.md]: Yes I think so, but I'll let your team write it 😄

Comment (benbarnett) [reply docs/components/stories.md]: 👍

**Pull request merged**